applications/nrf5340_audio/src/audio/audio_datapath.h

```diff
@@ -53,14 +54,9 @@ void audio_datapath_pres_delay_us_get(uint32_t *delay_us);
  *       and processed before being outputted over I2S. The audio is synchronized
  *       using sdu_ref_us
  *
- * @param buf Pointer to audio data frame
- * @param size Size of audio data frame in bytes
- * @param sdu_ref_us ISO timestamp reference from BLE controller
- * @param bad_frame Indicating if the audio frame is bad or not
- * @param recv_frame_ts_us Timestamp of when audio frame was received
+ * @param audio_frame Pointer to the audio data frame to be processed
  */
-void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref_us, bool bad_frame,
-			       uint32_t recv_frame_ts_us);
+void audio_datapath_stream_out(struct audio_data *audio_frame);
```

Comment:
Should this be 
```suggestion
void audio_datapath_stream_out(const struct audio_data *audio_frame);
```

Reply:
We might want to reuse the audio_frame to contain decoded audio_data, since most of the meta data should be the same, but can add a `const` for now.